app/components/form/instance-table.js

```diff
@@ -0,0 +1,61 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+import { inject as service } from '@ember/service';
+
+export default class InstanceTableComponent extends Component {
+  @service store;
+
+  @tracked formInfo = null;
+
+  get initialized() {
+    return this.formInfo !== null;
+  }
+
+  get editRoute() {
+    if (this.args.editRoute) {
+      return this.args.editRoute;
+    }
+    return 'form.instance';
+  }
+
+  @action
+  async removeInstance(instanceId) {
+    const result = await fetch(
+      `/form-content/${this.formInfo.form.definition.id}/instances/${instanceId}`,
```

Comment:
Maybe the route `/form-content` should be in a .env file?
Generally could be safer to have a helper method that puts the "/" for you, so you don't accidentally break the URI

Reply:
i'd avoid too many layers of indirection for now, the app is already quite complicated this would add to it for little gain imo